### PR TITLE
Consolidate tutorials in the docs and restyle

### PR DESCRIPTION
**Pull request by BarakHirshberg**

### CODE REVIEW

**Comment by BarakHirshberg** [comment]
@ceriottm @mahrossi @venkatkapil24 @litman90

**Comment by BarakHirshberg** [comment]
Hey guys, I am also done changing the format to furo. It looks much nicer.
Ready for review. @ceriottm @venkatkapil24 @litman90 @mahrossi

**Comment by venkatkapil24** [comment]
@mahrossi  @ceriottm @litman90  I am also using this branch to make the README more informative. I am thinking of moving the content of docs/README to the main README as currently there is no mention of i-PI's docs in the main README. Thought?

File: docs/src/onlinereso.rst [hunk 1]
```diff
@@ -1,7 +1,15 @@
 .. _librarywebsites:
 
-On-line resources
-=================
+Tutorials, recipes, and on-line resources
```
Comment (venkatkapil24) [inline docs/src/onlinereso.rst:3]: Would it also make sense to add a link to MLIPs with i-PI? https://github.com/venkatkapil24/MLIPs-with-iPI

**Review by venkatkapil24 — COMMENTED** [review]
This hopefully addresses the confusion in #398

File: docs/requirements.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ jinja2 < 3.1
 sphinxcontrib-bibtex==2.1.4
 numpy
 pathlib
+furo==2023.5.20
```
[the remark this answers is not in the document]
Comment (ceriottm) [reply docs/requirements.txt]: I removed the pin, and either with or without it works on my system and in the CI.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,9 @@
-i-PI: a Universal Force Engine
-==============================
+# i-PI: a Universal Force Engine
 
 A Python interface for ab initio path integral molecular dynamics simulations (and more).
```
Comment (ceriottm) [inline README.md:3]:
```suggestion
A Python interface for advanced molecular simulations, that can be combined with software performing energy and force evaluations based on first-principles calculations, empirical forcefields, and machine-learning  potentials.
```

**Review by ceriottm — DISMISSED** [review]
LGTM. I'd say merge and move on, we can always do a further round of polishing.

**Review by venkatkapil24 — DISMISSED** [review]

**Review by venkatkapil24 — APPROVED** [review]

**Pull request merged**